.gitignore

```diff
@@ -48,3 +48,17 @@ packages/**/*.map
 # cache
 .eslintcache
 
+# build files
+packages/**/lib
+packages/**/yarn.lock
+!packages/webpack-cli/lib
+
+# test output files
+test/js/*
+test/**/bin/
+test/**/**/bin/
+test/**/**/**/bin/
+test/**/**/binary/**
```

Comment:
Aren't these files get automatically cleaned after tests?

Reply:
yep they are but if you cancel your tests in mid then nope